src/drivers/batt_smbus/batt_smbus.h

```diff
@@ -48,12 +49,17 @@
 #include <mathlib/mathlib.h>
 #include <perf/perf_counter.h>
 #include <px4_platform_common/module.h>
+#include <px4_platform_common/param.h>
 #include <px4_platform_common/getopt.h>
 #include <px4_platform_common/i2c_spi_buses.h>
 #include <uORB/topics/battery_status.h>
 
 #include <board_config.h>
 
+using namespace time_literals;
+
+#define BATT_SMBUS_MEASUREMENT_INTERVAL_US              100_ms         ///< time in microseconds, measure at 4Hz
```

Comment:
```suggestion
#define BATT_SMBUS_MEASUREMENT_INTERVAL_US              100_ms         ///< time in microseconds, measure at 10Hz
```

Reply:
@dakejahl - thanks, fixed.
again, rebased for clearer log.

Reply:
@dakejahl - thanks, the note has fixed
the reason you see 4Hz there is because I suspect (from partial data on the datasheet) that the bq data is only relevant on 4Hz, so on my repo it does run @ 4Hz, But I didn't yet had time to check it on z50, or to really check the data update rate.
that for later checks